server/methods/accounts/accounts.app-test.js

```diff
@@ -47,6 +49,16 @@ describe("Account Meteor method ", function () {
   beforeEach(function () {
     shopId = getShop()._id;
     sandbox = sinon.sandbox.create();
+
+    fakeUser = Factory.create("user");
+    const userId = fakeUser._id;
+    // set the _id... some code requires that Account#_id === Account#userId
+    fakeAccount = Factory.create("account", { _id: userId, userId, shopId });
+    sandbox.stub(Meteor, "userId", () => userId);
+    sandbox.stub(Meteor, "user", () => fakeUser);
+    sandbox.stub(Reaction, "getShopId", () => shopId);
+
+    Object.keys(originals).forEach((method) => spyOnMethod(method, userId));
```

Comment:
no need to do this in each spec. doing it here in the beforeEach works, since we know the current userId